packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -1228,33 +1482,48 @@ describe('ReactErrorBoundaries', () => {
       // Render the error message
       'ErrorBoundary componentWillUpdate',
       'ErrorBoundary render error',
+      'ErrorBoundary componentWillReceiveProps',
```

Comment:
I added these expected log statements to make the tests pass I know, `createRoot(..).render` is more efficient than `ReactDOM.render` so I assumed this is a byproduct of these functional differences but that could be wrong.